src/queues/sendMessage.ts

```diff
@@ -17,15 +18,40 @@ declare global {
   }
 }
 
-const processJob: JobProcessor<typeof name> = () => async (job) => {
-  const { message, key } = job.data;
+const MESSAGE_LINES_LIMIT = 10;
+const breakDiscordMessage = (message: string) => {
+  const lines = message.split('\n');
+  if (lines.length <= MESSAGE_LINES_LIMIT) return { current: message };
```

Comment:
i would take while we are under the character limit instead

Reply:
updated 🙏